### PR TITLE
issue #274

### PR DESCRIPTION
**Pull request by KurtStam**
Splitting DataShape.dataType up into kind and type

### CODE REVIEW

**Comment by KurtStam** [comment]
@gashcrumb Does this look out you?

**Comment by gashcrumb** [comment]
@KurtStam +1 though what does `'*'` vs `'none'` mean I wonder...

**Comment by jimmidyson** [comment]
@gashcrumb Perhaps `any` vs `none` would make more sense?

**Comment by gashcrumb** [comment]
@jimmidyson yeah, that's more self-describing I think.

File: dao/src/main/resources/com/redhat/ipaas/dao/deployment.json [hunk 1]
```diff
@@ -244,10 +246,10 @@
           "name": "HTTP GET",
           "description": "Call a service that is internal (within your company) or external (on the internet) by specifying the service's URL",
           "inputDataShape": {
-             "dataType" : "none"
+             "kind" : "none"
           },
           "outputDataShape": {
-             "dataType" : "*"
+             "kind" : "*"
```
Comment (jimmidyson) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: Can we use `any` instead of `*` please?

**Comment by KurtStam** [comment]
@gashcrumb 

`none` means there is not data going in/out. Like twitter there in no data going 'in', only 'out', so dataShapeIn is `none`. Would you prefer it not set?

`*` or soon to be `any`refers that it is free form. Like for the HTTP GET connector it can be anything.

**Comment by gashcrumb** [comment]
let's leave it as `'none'` I think.

**Comment by KurtStam** [comment]
@jimmidyson I think we are good to go on this. Can you approve & merge plz?

**Pull request merged**